packages/ui/src/lib/table/Table.spec.ts

```diff
@@ -380,3 +380,20 @@ test('Expect duration cell to be empty when undefined', async () => {
     expect(cells[6].textContent?.trim()).toBe(expected[i - 1]);
   }
 });
+
+test('Expect table is scoped for css manipulation', async () => {
+  render(TestTable, {});
+
+  // Wait for the table to update
+  await tick();
+
+  const table = await screen.findByRole('table');
+  expect(table).toBeDefined();
```

Comment:
nit: to be sure that changes does not affect table rendering there should be check for `table.style.gridTemplateColumns` not an empty string.

Reply:
test updated